packages/itwin/grouping-mapping-widget/src/grouping-mapping-widget.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** UI Provider for iTwin Viewer Applications */
+export * from "./widget/GroupingMappingWidget";
+
+/** Internal components for custom UIs */
+export * from "./api/generated";
```

Comment:
not sure how i feel about this. we need to talk later about if this should be pulled out into its own client pkg like all the others.

Reply:
do we really want to be exporting the entire generated api?

Reply:
Nope not needed anymore.